packages/ag-charts-community/src/chart/interaction/syncManager.ts

```diff
@@ -0,0 +1,44 @@
+import type { Chart } from '../chart';
+import { BaseManager } from './baseManager';
+
+type GroupId = string | symbol;
+
+export class SyncManager extends BaseManager {
+    static chartsGroups = new Map<GroupId, Set<Chart>>();
+    static DEFAULT_GROUP = Symbol('sync-group-default');
```

Comment:
Minor nit: Any reason to not just make this a `const` outside the class definition?

Reply:
:+1: